src/containers/Tenant/Diagnostics/TopShards/TopShards.tsx

```diff
@@ -190,53 +191,60 @@ export const TopShards = ({tenantPath, type}: TopShardsProps) => {
             };
         };
 
-        const columns: Column<any>[] = [
+        const columns: Column<KeyValueRow>[] = [
             {
                 name: tableColumnsNames.Path,
-                render: ({value: relativeNodePath}) => {
+                render: ({row}) => {
```

Comment:
Changed `value` to `row`, because `value` is of `unknown` type while `row` is typed with an argument to `Column` (`KeyValueRow` in this case)